tekton/cronjobs/prow-config-cd-hourly-cron/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+bases:
+- ../resource-cd-cron-base
```

Comment:
one thing i dont quite understand (but im totally new to kustomize!): does it make sense to have a base when there is only one thing overriding it? could we get away with a simple config file until we need to override?

(is override the right word? overlay?)

Reply:
I guess so. I started with the idea of having only a cronjob types like one for image build, one for CD of something, one for releases. Also the cronjob with the git sha and build uuid looked complex enough that I wanted to make sure it was re-used - but I ended up not re-using it... yet. That's something that should be fixed probably.

One way to simplify cronjobs could be to inject always a standard set of parameters for everyone, and then use an interceptor to add anything that might be missing for specific pipelines. But then we would have to run this interceptor and maintain its config and CD that too... so I'm not sure which option is best

Reply:
coming back to this way later and having forgotten most of the context :sweat_smile: : i still would like to avoid having a "base" when we don't need one if possible

Reply:
Once this is merged, I was planning on adding more auto-deployed config using the same base, which is why I wanted to leave this in. I could move this to the next PR if you prefer so.